migration/migrator/migrations/master/20231106203826_superuser_announcement.py

```diff
@@ -0,0 +1,55 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+    # Create the sequence for superuser_announcements id
+    database.execute(
+    """
```

Comment:
It would be preferable to have the database.execute be only called once, which would mean saving the sequel code in a string and then calling the execute function